### PR TITLE
Card errors (wallet lock): Reject and reset USB commands

### PR DESCRIPTION
**Pull request by ujjwal-cyph**
Fixes https://app.clickup.com/t/37308523/CHI-2038

### CODE REVIEW

**Comment by vbcypherock** [comment]
Task linked: [CHI-2038 Fix comm error after Card errors](https://app.clickup.com/t/37308523/CHI-2038)

File: src/level_four/tap_cards/controller/controller_tap_cards.c [hunk 1]
```diff
@@ -219,10 +219,13 @@ bool tap_card_handle_applet_errors() {
                 if (tap_card_data.status == SW_NO_ERROR)
                     add_challenge_flash((const char *) wallet.wallet_name, target, random_number,
                                         tap_card_data.tapped_card);
-                tap_card_data.active_cmd_type = USER_ENTERED_PIN;
                 tap_card_data.lvl3_retry_point = WALLET_LOCKED_MESSAGE;
                 flow_level.level_two = LEVEL_THREE_WALLET_LOCKED;
                 decrease_level_counter();
+                if (tap_card_data.desktop_control) {
+                    comm_reject_request(USER_ENTERED_PIN, 0);
```
Comment (ujjwal-cyph) [inline src/level_four/tap_cards/controller/controller_tap_cards.c:226]: Use the existing command id for locked wallet

```suggestion
                    comm_reject_request(WALLET_IS_LOCKED, 0);
```

File: src/controller_main.c [hunk 2]
```diff
@@ -209,6 +209,7 @@ void reset_flow_level()
 {
     //reset device state
     CY_Reset_Not_Allow(true);
+    CY_Set_External_Triggered(false);
```
Comment (amanCypherock) [inline src/controller_main.c:212]: Just need to test this change for all flows, in case it has an impact

Comment (ujjwal-cyph) [reply src/controller_main.c:212]: completed

**Pull request merged**